src/indexes.rs

```diff
@@ -82,6 +82,46 @@ impl<'a> Index<'a> {
         ).await?)
     }
 
+    /// Delete the index if it exists.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use meilisearch_sdk::{client::*, indexes::*};
+    /// # futures::executor::block_on(async move {
+    /// let client = Client::new("http://localhost:7700", "masterKey");
+    /// # client.create_index("movies", None).await;
+    ///
+    /// // get the index named "movies" and delete it
+    /// let movies = client.assume_index("movies");
+    /// let mut deleted = movies.delete_if_exists().await.unwrap();
+    /// assert_eq!(deleted, true);
```

Comment:
It may be strange for somebody reading this example because `deleted` could be `false` if the line 93 was not here, and the line 93 is hidden in the doc. Consider hiding this line too or making line 93 visible.